### ISSUE TITLE
New User Improvements: Update landing page

### ISSUE THREAD

**Issue by laurenliz22** [SEED-platform/seed#2592]
Testing

**Comment by laurenliz22** [comment]
@RDmitchell - The following New User Improvement tickets are suggestions our team developed to enhance the experience for new SEED Platform users.  Before we start working through these tickets to update SEED we want to get your sign off on the changes.  Would you mind looking at the descriptions and letting me know if you have any questions or changes.  Thanks! Lauren

**Comment by RDmitchell** [comment]
@laurenliz22 -- I think most of the suggestions make sense, except for the following

* I would keep the name Data and not change it to Datasets
* I'm not sure changing Inventory to Properties makes sense, although maybe it does. When you get to the Inventory page it has a tab for both Properties and Tax Lots, which is why it seems like a more inclusive name than just Properties would be better. But on the other hand, users are generally entering properties into the software and tax lots are generally secondary information. This might warrant a bit more discussion. I could potentially ask a few existing users what they think. And probably it really doesn't matter because users just get used to what is there. 
* The issue of a user having to make an organization after the fact is something I don't know about, as I don't know how the new way that people can make their own accounts work. It seems like it would be better to have the user make an organization when they create their initial account but I'm sure there is a good reason (that I don't know about) to do it the way you are planning to do it.

**Comment by laurenliz22** [comment]
@RDmitchell thank you for looking through these!  A couple of notes:
- We thought that "Data" and "Inventory" may be confusing for new users since technically the "Inventory" tab also shares the building data, just in an expanded format.  We thought that using "Properties" instead would be more clear in describing what shows up on the list.  The first tab that the user will see is the "Property" tab, so we thought this may make sense.  However, I have a meeting with the group today and I'll gather some more feedback.  
- The creation of an organization is something that we are currently working on in another ticket.  I believe when a new user signs in for the first time they will be prompted to create an organization to allow them to use the site, if they do not have an existing organization they are logging into.  This message will just be a reminder to them to do so.

**Comment by laurenliz22** [comment]
@RDmitchell I was wondering if there are names that you would prefer for the "Data" and "Inventory" pages that may better explain the content?  Any suggestions would be very helpful.  Thanks!

**Comment by RDmitchell** [comment]
Changing Inventory to Properties is probably fine. 

I don't have any better ideas. 

Not sure Datasets is any clearer than Data, however, so I'm not sure it's worth changing that.

**Comment by Ryoken** [comment]
Hey @RDmitchell, what are your thoughts?

**Comment by RDmitchell** [comment]
This seems fine. I guess my only real comment is that it's hard to read the text in the middle of the screen because of the very detailed photo in the background. Not sure what to do about that, though.

**Comment by Ryoken** [comment]
@RDmitchell We've tweaked it a little to make the background darker and less visual, as well as the text clearer.  Is this an improvement?

<img width="1301" alt="Screen Shot 2021-03-26 at 3 25 51 PM" src="https://user-images.githubusercontent.com/564231/112694312-c393d600-8e47-11eb-9f89-eae0ca1c82df.png">

**Comment by RDmitchell** [comment]
This looks good. Thx
